keyboards/handwired/elrgo_s/readme.md

```diff
@@ -0,0 +1,17 @@
+# Elrgo S Split Keyboard
+
+This is a fork of the VOID Ergo S with QMK Configurator support.
+
+Elrgo S uses 2x Arduino Pro Micro (ATmega32U4) with 3D printed case.
+ 
+[Building guide, models for 3D printing](https://github.com/Eloren1/Elrgo_S)
+
+* Keyboard Maintainer: [Eloren](https://github.com/Eloren1)
+* Hardware Supported: Arduino Pro Micro
+* Hardware Availability: [Open Source](https://github.com/Eloren1/Elrgo_S).
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to enter the bootloader, to flash new firmware?